lib/deprecation-cop-status-bar-view.coffee

```diff
@@ -34,13 +34,16 @@ class DeprecationCopStatusBarView extends View
 
   getDeprecatedSelectorCount: ->
     @deprecatedSelectorCount ?= getSelectorDeprecationsCount()
+  
+  getDeprecatedCallCount: ->
+    (deprecation.getStacks().length for deprecation in Grim.getDeprecations()).reduce(((a, b) -> a + b), 0);
```

Comment:
> Just wondering if this might be more straight-forward as:

Definitely :+1:. I think this was the first time I used a list comprehension in CoffeeScript and wished there was a better way to do this. :smile: Using that approach as of e3d0642d0f7ad60eb6c042d66000df4dff330cd0.